ocfweb/docs/templates/docs/officers.html

```diff
@@ -142,6 +143,24 @@ <h5>Deputies:</h5>
               {% endif %}
             </td>
           </tr>
+          {% if term.heads %}
+            <tr>
+              <td style="border: none;" />
+              <td colspan="2">
+                <details>
+                  <summary style="display: list-item;">
+                    Committee Heads
+                  </summary>
+                  {% for committee in term.heads %}
+                    <h5>{{ committee.name }} Heads</h5>
```

Comment:
Probably don't need the word "Heads" on this line, the word head is used a lot

Reply:
I am worried if we do not put "Heads" here then people will think the heads are the whole committee.

Reply:
The word is still used in the dropdrown menu so I don't think it'll have that effect

Reply:
ok, done